masakari-controller/controller/masakari_controller.py

```diff
@@ -47,10 +46,14 @@
 import controller.masakari_starter as starter
 from controller.masakari_util import RecoveryControllerUtil as util
 from controller.masakari_util import RecoveryControllerUtilDb as util_db
+from controller.masakari_util import LogProcessBeginAndEnd
+from oslo_log import log as oslo_logging
 import controller.masakari_config as config
 import controller.masakari_worker as worker
 import db.api as dbapi
 
+log_process_begin_and_end = LogProcessBeginAndEnd()
```

Comment:
If you use instance method as a decorator we should pass logger defined following line (or \__name__\)  to the instance and use the logger to output messages.  It's hard to read the message since the debug message could be following.

    controller.masakari_util BEGIN method-name: parameter is () {}
    controller.masakari_controller log message1 from method-name
    controller.masakari_controller log message2 from method-name
    controller.masakari_util END method-name: retturn